ThePortugueseMan.HabitTracker/HabitsLibrary/MainTable.cs

```diff
@@ -0,0 +1,44 @@
+﻿using DataBaseLibrary;
+using Microsoft.Data.Sqlite;
+using System;
+
+namespace HabitsLibrary;
+
+public class MainTable
+{
+    private static DataBaseCommands dbCommands = new();
+    public string? tableName;
+    public string? connectionString;
+
+    public MainTable(string tableName, string connectionString) 
+    {
+        this.tableName = tableName;
+        this.connectionString = connectionString;
+    }
```

Comment:
Please add empty line between all methods. And remove empty lines after opening curly brackets and before closing curly brackets. See code conventions here: https://thecsharpacademy.com/article/58